tests/integration/test_types_integration.py

```diff
@@ -284,13 +288,15 @@ def test_time(trino_connection):
 
 
 @pytest.mark.skipif(trino_version() == '351', reason="time not rounded correctly in older Trino versions")
-def test_time_with_timezone(trino_connection):
-    query_time_with_timezone(trino_connection, '-08:00')
-    query_time_with_timezone(trino_connection, '+08:00')
-    query_time_with_timezone(trino_connection, '+05:30')
-
-
-def query_time_with_timezone(trino_connection, tz_str: str):
+@pytest.mark.parametrize(
```

Comment:
Same logic as before but more aligned with how `pytest` was intended to be used.